### PR TITLE
feat:  add active status to navbar menu Group

### PR DESCRIPTION
**Pull request by casyalex**
add active style to VPFlyout, make navbar group can show active status.

### CODE REVIEW

File: docs/.vitepress/config.ts [hunk 1]
```diff
@@ -44,11 +44,15 @@ export default defineConfig({
...
-    { text: 'Guide', link: '/guide/what-is-vitepress', activeMatch: '/guide/' },
+    // { text: 'Guide', link: '/guide/what-is-vitepress', activeMatch: '/guide/' },
     { text: 'Configs', link: '/config/introduction', activeMatch: '/config/' },
     {
       text: 'Changelog',
       link: 'https://github.com/vuejs/vitepress/blob/main/CHANGELOG.md'
+    },{
+      text:'Gruop Nav',  
+      activeMatch: '/guide/',
+      items: [{ text: 'Guide', link: '/guide/what-is-vitepress', },]
```
Comment (kecrily) [inline docs/.vitepress/config.ts:55]: The changes made during testing should be reverted.

File: src/client/theme-default/components/VPFlyout.vue [hunk 2]
```diff
@@ -23,20 +23,9 @@ function onBlur() {
...
-      type="button"
-      class="button"
-      aria-haspopup="true"
-      :aria-expanded="open"
-      :aria-label="label"
-      @click="open = !open"
-    >
+  <div class="VPFlyout" ref="el" @mouseenter="open = true" @mouseleave="open = false">
+    <button type="button" class="button" aria-haspopup="true" :aria-expanded="open" :aria-label="label"
+      @click="open = !open">
```
Comment (kecrily) [inline src/client/theme-default/components/VPFlyout.vue:28]: Here too it seems to be a mere change of style and they should not be modified.

**Review by kecrily — CHANGES_REQUESTED** [review]

**Comment by casyalex** [comment]
useless change removed. Thanks

**Comment by kiaking** [comment]
Thanks a lot! 🙌

**Pull request merged**